var/spack/repos/builtin/packages/python/package.py

```diff
@@ -36,7 +36,7 @@ class Python(Package):
     list_url = "https://www.python.org/ftp/python/"
     list_depth = 1
 
-    maintainers = ["adamjstewart", "skosukhin", "scheibelp", "varioustoxins", "pradyunsg"]
+    maintainers = ["adamjstewart", "skosukhin", "scheibelp", "pradyunsg"]
```

Comment:
@varioustoxins if I remember correctly, your willingness to help maintain this build recipe was contingent on Spack having Python 2 support. I removed you since we're planning on removing Python 2 support, but let me know if you still want to be listed as a maintainer.